### ISSUE TITLE
dita -install does not work (DITA-OT 2.1.2)

### ISSUE THREAD

**Issue by xephon2** [dita-ot/dita-ot#2182]
The `dita -install myplugin.zip` command does not work with DITA-OT 2.1.2. It works using the DITA-OT 2.2.1.

What I did:

1. Launch `startcmd.bat`.
2. `cd bin`
3. `dita -install myplugin.zip`

No error message appears. Maybe this is related to #2002.

**Comment by jelovirt** [comment]
One thing that may or may not be related: if you use the `dita` command, do *not* use the `startcmd.bat`. Instead, open a new Command Prompt from start menu.

**Comment by jelovirt** [comment]
Could you rerun the integration with `-d` argument to get some more logging?

**Comment by xephon2** [comment]
Opening a command line interface my launching `cmd.exe` makes no difference. The `-d` parameter does not help.

```
C:\Users\eike\DITA-OT\dita-ot-2.1.2\bin>dita -d -install com.oxygenxml.webhelp.zip
Unable to locate tools.jar. Expected to find it in C:\Program Files\Java\jre1.8.0_66\lib\tools.jar
Searching for integrator.xml ...
```

**Comment by jelovirt** [comment]
If this works in 2.2 but not in 2.1, I'm somewhat inclined to mark this as won't fix, as we don't have the resources to maintain older versions. @robander what's you're take.

**Comment by xephon2** [comment]
I'd suggest that the DITA-OT should have Long Term Support versions, that will receive some picked cherrys for at least 12 months or so. Our plugins are incompatible with 1.8.5, but each new 2.x version closes some bugs and ships new ones. Or you should explain, that the OT is a rolling release tool, that has to be updated by vendors in real time, like a browser. LTS is probably what the vendort want.

**Comment by jelovirt** [comment]
LTS is what vendors want, but because no vendor is willing to do the work, we're stuck with the resources we currently have. OT is not a rolling version number since we don't release e.g. every 6 weeks. However, due to resource limits, we can't really update versions other than the latest release (2.2 as of writing this). When we relese 3.x, we will probably update 2.x for a while. But, the support will not last that long and all resources are directed to the latest development.

**Comment by jelovirt** [comment]
Closing as we currently don't intend to release new bug fixes in 2.1 branch.

**Comment by robander** [comment]
Late to this as I was out of the office, but agree to what @jelovirt said above. I'd love to have LTS for at least the last couple of major releases, but with the current team that's not possible. I think we carried on releasing fixes for 1.8.x a bit longer than usual when 2.0 came out, and that's probably going to be the same for the last major 2.x when we  come out with a 3.0. Otherwise, we're limited by the current resources.
